src/System.Runtime.Extensions/tests/VoidMainWithExitCodeApp/project.json

```diff
@@ -7,6 +7,7 @@
     "System.Threading.Thread": "4.3.0-beta-devapi-24503-01"
   },
   "frameworks": {
+    ".NETCoreApp,Version=v1.0": {},
```

Comment:
Actually why did you need to add these at all? I don't see any new build configurations in this PR that would need them.